pep_sphinx_extensions/pep_theme/static/style.css

```diff
@@ -177,6 +177,11 @@ sup {top: -0.5em}
 sub {bottom: -0.25em}
 
 /* Table rules */
+div.table-wrapper {
+    overflow-x: scroll;
+    scrollbar-width: thin;  /* CSS Standards, not *yet* widely supported */
```

Comment:
Indeed, only supported by Firefox: https://caniuse.com/?search=scrollbar-width

MDN also notes accessibility concerns, `scrollbar-width: thin` "can make content hard or impossible to scroll if the author does not provide an alternative scrolling mechanism": https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-width#accessibility_concerns

As we're doing this partly for accessibility shall we skip this one?

Reply:
Yeah, I wanted to be consistent with the other scroll bars (e.g. for the sidebar), but that also uses the illegal, bespoke pseudo-elements to support WebKit/Blink-based browsers, so you're right—best to just eliminate this, fall back on the device-default scrollbars and use `overflow-x: auto` instead to automatically hide it when not needed instead of a transparent overlay. I also fixed several other additional existing responsiveness issues I spotted in further testing for this.